python/ray/air/checkpoint.py

```diff
@@ -199,6 +199,38 @@ def __repr__(self):
         parameter, argument = self.get_internal_representation()
         return f"{self.__class__.__name__}({parameter}={argument})"
 
+    @property
+    def uri(self) -> Optional[str]:
```

Comment:
Mention in the doc that you can restore the checkpoint from the URI with `Checkpoint.from_uri`?

Reply:
Since we are working with a Checkpoint object already when we're calling `.uri`, this is a  it tautological, so I left it out